pkg/graph/resolvers/itgovactions/lcidactions/update_lcid.go

```diff
@@ -0,0 +1,85 @@
+package lcidactions
+
+import (
+	"time"
+
+	"github.com/guregu/null"
+
+	"github.com/cmsgov/easi-app/pkg/apperrors"
+	"github.com/cmsgov/easi-app/pkg/models"
+)
+
+// GetUpdateLCIDAction is a method to set all the relevant data fields for that are a result of an Update LCID action.
+func GetUpdateLCIDAction(
+	intake models.SystemIntake,
+	expirationDate *time.Time,
+	nextSteps *models.HTML,
+	scope *models.HTML,
+	costBaseline *string,
+	userInfo models.UserInfo,
+) models.Action {
+
+	action := getBaseLCIDAction(intake, expirationDate, nextSteps, scope, costBaseline, userInfo)
+	action.ActionType = models.ActionTypeUPDATELCID
+
+	return action
+
+}
+
+// IsLCIDValidToUpdate checks if you can update the LCID for an intake, if not, it will return an error
+// An LCID must have already been issued in order to be updateable
+func IsLCIDValidToUpdate(intake *models.SystemIntake) error {
+	if !intake.LifecycleID.Valid {
+		return &apperrors.BadRequestError{
+			Err: &apperrors.InvalidActionError{
+				ActionType: models.ActionTypeUPDATELCID,
+				Message:    "an LCID has not been issued for this intake, an LCID must be issued before it can be updated",
+			},
+		}
+	}
+	return nil
+}
+
+// getBaseLCIDAction returns the basic entries for an aciton. Action type is handled by the calling action
```

Comment:
Nitpick - this might be more clear as stating that it returns an action with the "basic entries" set, or that it prepares an action, or something.

Reply:
I update the comment to be clearer.